functions/Test-DbaDbCompression.ps1

```diff
@@ -40,11 +43,34 @@ function Test-DbaDbCompression {
     .PARAMETER ExcludeDatabase
         The database(s) to exclude - this list is autopopulated from the server
 
+    .PARAMETER Schema
+        Filter to only get specific schemas If unspecified, all schemas will be processed.
+
+    .PARAMETER Table
+        Filter to only get specific tables If unspecified, all User tables will be processed.
+
+    .PARAMETER MaxResultsPerDb
```

Comment:
I think we can just call this one `ResultSize` or maybe just even `Rows`?

Reply:
Will change to ResultSize. Using Rows may be misleading as, depending on the FilterBy setting, this will mot be strictly accurate.

Reply:
Will change to ResultSize. Using Rows may be misleading as, depending on the FilterBy setting, this will mot be strictly accurate.